PrivateCollections/Germany/Neukirchen006.xml

```diff
@@ -0,0 +1,502 @@
+              </msIdentifier>
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="5r" to="143r"/>
+                     <title type="complete" ref="LIT2701Dawit"/> 
+                     <textLang mainLang="gez"/>
+
+                    <msItem xml:id="ms_i1.1">
+                      <locus from="5r" to="110r"/>
+                      <title type="complete" ref="LIT2000Mazmur"/>
+                      <textLang mainLang="gez"></textLang>
+                      <incipit xml:lang="gez"><hi rend="rubric">
+                        ዘከመ᎓ መዝሙረ፡ ድንግል፡ ትመስል፡ ገነተ።</hi>
+                        ወታስተጋብእ᎓ አቅማኃ᎓ ወፍሬያት፨
+                        <hi rend="rubric">ወትሰድድ᎓ አጋንንተ᎓ ወታቄርብ᎓ መላእክተ።</hi>
+                         ተግሣጽ᎓ ለኵሉ᎓ ወአሥተናሥኦ᎓ ምግባራት፨፨፨
+                         <hi rend="rubric">
+                          ፍካሬ᎓ ዘጻድቃን᎓ ወዘኃጥአን።
+                          መዝሙር᎓ ዘዳዊት᎓ ሃሌሉያ።
+                      ፩</hi>ብፁዕ᎓ ብእሲ᎓ ዘኢሖረ᎓ በምክረ᎓ ረሲዓን፨
+                    ወዘኢቆመ᎓ ውስተ᎓ ፍኖተ᎓ ኃጥአን፨
+                     <hi rend="rubric">ወዘኢነበረ᎓ ውስተ᎓ መንበረ᎓ መስተሣልቃን።</hi></incipit>
+                     <explicit xml:lang="gez">ወአሰሰልኩ᎓ ጽእለተ᎓ እምደቂቀ᎓ እስራኤል፨
+                       <add place="below">ወአንሰ᎓ ነሣእኩ᎓ እምውስተ᎓ አፍላግ᎓ ፫አዕባነ፨</add>
+                   </explicit>
+                      </msItem>
+
+                    </msItem>
+                    <msItem xml:id="ms_i1.2">
+                      <locus from="110v" to="121v"/>
+                      <title type="complete" ref="LIT1828Mahale">
+                        <hi rend="rubric">መሐልየ᎓ ነቢያት᎓ ወጸሎቱ᎓ ለሙሴ።</hi></title>
+                      <incipit xml:lang="gez">
+                       <hi rend="rubric">ንሴብሖ᎓ ለእግዚአብሔር᎓ ስቡሕ᎓ ዘተሰብሐ።</hi>
+                     ፈረሰ᎓ ወመስተፅዕነ᎓ ወረወ᎓ ውስተ᎓ ባሕር፨</incipit>
+                       <explicit xml:lang="gez">እስመ᎓ ርእያ᎓ አዕይንትየ᎓ አድኅኖተከ፨
+                       ዘአስተዳሎከ᎓ ቅድመ᎓ ኵሉ᎓ ሕዝብከ፨
+                        ከመ᎓ ትክሥት᎓ ብርሃነ᎓ ለአሕዛብ፨
+                      ወክብረ᎓ ለሕዝብከ᎓ እስራኤል፨</explicit>
+                    </msItem>
+
+                    <msItem xml:id="ms_i1.3">
+                      <locus from="121v" to="127v"/>
+                      <title type="complete" ref="LIT2362Songof">መሐልየ፡ መሐልይ᎓ ዝውእቱ᎓ ዘሰሎሞን።</title>
+                    <incipit xml:lang="gez"><hi rend="rubric">ይስዕመኒ᎓ በስዕመተ᎓ አፉሁ።</hi>
+                   አዳም᎓ አጥባተኪ᎓ እምወይን፨
+                 </incipit>
+                     <explicit xml:lang="gez">ጕይ᎓ አንተ᎓ ወልድ᎓ እኁየ᎓ ወተመሰላ᎓ ለወይጠል፨
+                     ወከመ᎓ ወሬዛ᎓ ኃየል᎓ ውስተ᎓ አድባራተ᎓ ድኍኃን፨</explicit>
+                    </msItem>
+
+                    <msItem xml:id="ms_i1.4">
+                      <locus from="128ra" to="138va"/>
+                      <title type="complete" ref="LIT2509Weddas"/>
+                      <textLang mainLang="gez"/>
+                      <incipit xml:lang="gez"><hi rend="rubric">ውዳሴሃ፡ እግዚእትነ᎓ ማርርያም᎓
```

Comment:
ማርርያም᎓ in the manuscript? Then you could mark it up with sic, to avoid confusion